lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnContainerManager.java

```diff
@@ -535,13 +537,28 @@ private void handleNewContainer(final Container container) {
       LOG.log(Level.FINEST, "Allocated Container: memory = {0}, core number = {1}",
           new Object[] {container.getResource().getMemory(), container.getResource().getVirtualCores()});
 
+      YarnClient client = YarnClient.createYarnClient();
```

Comment:
Can you add a comment saying TODO ISSUE-NUMBER: update this part when YARN supports a method to retrieve the node label expression of a container and create a JIRA issue? 
Make sure you follow the exact TODO format.